persistence/dynamodb/index.md

```diff
@@ -55,6 +55,22 @@ and registered on the container
 
 snippet: DynamoDBCustomClientProviderRegistration
 
+## Permissions
+
+Below is the list of minimum required [IAM policies for operating the DynamoDB persistence](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/security_iam_service-with-iam.html) with installers enabled: 
+
+  - `dynamodb:CreateTable`,
+  - `dynamodb:DescribeTable`,
+  - `dynamodb:DescribeTimeToLive`,
+  - `dynamodb:UpdateTimeToLive`,
+  - `dynamodb:Query`,
+  - `dynamodb:GetItem`,
+  - `dynamodb:BatchWriteItem`,
+  - `dynamodb:PutItem`,
+  - `dynamodb:DeleteItem`
+
+If installers are disabled, or if `DisableTableCreation` is called, the `dynamodb:CreateTable` and `dynamodb:UpdateTimeToLive` policies are not required. 
```

Comment:
I think DescribeTable is also only required when isntallers are enabled.

Reply:
Then we probably need to update our internal documentation too

Reply:
You are right. It is only used in the installer. Good catch

Reply:
Update the internal documentation too